build/package/circleci/orb.yml

```
@@ -69,11 +69,11 @@ jobs:
         type: string
         default: "https://app.launchdarkly.com"
       context_lines:
-        description: The number of context lines above and below a code reference for the job to send to LaunchDarkly. If < 0, no source code will be sent to LaunchDarkly. If 0, only the lines containing flag references will be sent. If > 0, will send that number of context lines above and below the flag reference. A maximum of 5 context lines may be provided.
-        type: string
-        default: "-1"
+        description: The number of context lines above and below a code reference for the job to send to LaunchDarkly. By default, the flag finder will not send any context lines to LaunchDarkly. If < 0, no source code will be sent to LaunchDarkly. If 0, only the lines containing flag references will be sent. If > 0, will send that number of context lines above and below the flag reference. A maximum of 5 context lines may be provided.
```

This is a little confusing. a possible improvement:
```suggestion
        description: The number of context lines above and below a code reference for the job to send to LaunchDarkly. Defaults to -1. If < 0, no source code will be sent to LaunchDarkly. If 0, only the lines containing flag references will be sent. If > 0, will send that number of context lines above and below the flag reference. A maximum of 5 context lines may be provided.
```

---

just noticed the default is also right there in the config (line 74). Do orb consumers see this?